drivers/sensor/ntc_thermistor/ntc_thermistor_calc.c

```diff
@@ -97,17 +97,17 @@ static void ntc_lookup_comp(const struct ntc_type *type, unsigned int ohm, int *
  * ntc_get_ohm_of_thermistor() - Calculate the resistance read from NTC Thermistor
  *
  * @cfg: NTC Thermistor configuration
- * @max_adc: Max ADC value
- * @raw_adc: Raw ADC value read
+ * @sample_mv: Measured voltage in mV
  */
-uint32_t ntc_get_ohm_of_thermistor(const struct ntc_config *cfg, uint32_t max_adc, int16_t raw_adc)
+uint32_t ntc_get_ohm_of_thermistor(const struct ntc_config *cfg, int sample_mv)
```

Comment:
This is looking more like [linux implementation](https://github.com/torvalds/linux/blob/master/drivers/hwmon/ntc_thermistor.c#L382).  Lets add the additional safety checks here as well.

Reply:
Done